### ISSUE TITLE
[Enhancement] Show Related Events Count within a MISP Event

### ISSUE THREAD

**Issue by jdubp** [MISP/MISP#2650]
Currently when you are within a MISP event, in the upper right under Related Events, it will show all the events that contain correlations with the current event. Would it be plausible to add a count in parenthesis next to Related Events (or somewhere else on the event page)? This would be helpful at a quick glance for events with a large number of correlations.

*If you would like to report a bug, please fill the template bellow*

### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug, Question, Feature Request, support...
| OS version (server)       | Debian, ubuntu, CentOS, RedHat, ...
| OS version (client)       | XP, Seven, 10, Ubuntu, ...
| PHP version               | 5.4, 5.5, 5.6, 7.0, 7.1...
| MISP version / git hash   | 2.4.XX, hash of the commit
| Browser                   | If applicable

**Comment by iglocska** [comment]
Yeah we need that for sure, on highly correlated instances it's messy as hell.
